src/common/wallet/walletManager.js

```diff
@@ -285,6 +291,28 @@ class WalletManager {
     }
     return null;
   }
+
+  createReadOnlyWallet = async (chain, type, address, coins) => {
+    console.log('walletManager.createReadOnlyWallet, coins', coins);
+
+    // 2. Create a Wallet instance and save into wallets
+    const wallet = ReadOnlyWallet.create({
+      id: this.currentKeyId, name: null, chain, type, address, coins,
+    });
+
+    console.log(`createReadOnlyWallet, address: ${address}`);
+    this.wallets.unshift(wallet);
+
+    // Increment current pointer
+    this.currentKeyId += 1;
+
+    // Save to storage
+    await this.serialize();
+
+    return wallet;
+  }
+
+  getNormalWallets = () => _.filter(this.wallets, { walletType: definitions.WalletType.Normal });
```

Comment:
Add comment to this function.

Instead of getNormalWallets() it's more scalable to make 

getWallets(WalletType.Normal) so you can get ReadOnly wallets with the same function.

Why WalletType is not defined in constants?
And why constants is a JSON. It's easier if it's a .js. Then you can do `import {WalletType} from './constants'`

Reply:
Fixed